### ISSUE TITLE
Can buy dev card on not my turn (Phase 3, Team 6)

### ISSUE THREAD

**Issue by matthewfcarlson** [thyer/CS340winter2016#575]
It looks like you can click on the modal and the thing comes up but based on the server output it isn't actually buy one.
<img width="1372" alt="screen shot 2016-03-25 at 5 03 45 pm" src="https://cloud.githubusercontent.com/assets/6557490/14055968/e2cfdb5a-f2ab-11e5-989e-e1733ca585c2.png">

**Comment by KasiKarra** [comment]
I'm confused.  So you are saying that you can click the buttons, but it doesn't actually go to the server so this would be a roll over phase 2 type error.  (Yes I know phase 2 errors are still valid, but I want to verify I get what you are talking about.

**Comment by matthewfcarlson** [comment]
Right. You can click the buy Dev card button and even click buy but nothing happens and the server doesn't show any log messages for handling the move.

**Comment by matthewfcarlson** [comment]
This is like p3 maybe p4
